lib/galaxy/config/sample/job_conf.xml.sample_advanced

```diff
@@ -385,6 +385,15 @@
                  (db-skip-locked, db-transaction-isolation) and the value is an integer > 0. Default is to grab as many
                  jobs ready to run as possible.
 
+               - `ready_window_size` - Handlers query for and dispatch jobs ready to run in a loop. If the number of
```

Comment:
xref https://github.com/galaxyproject/galaxy/pull/11228#discussion_r574016616

We should stop updating this file and update the YAML version instead.

Reply:
While I agree, we need to update all of the documentation and move the YAML over to the sample dir, update the Ansible role, etc. AFAIK I am the only person using it so it seems a bit premature to dump the XML without at least having the docs to help people transition.

Reply:
Let me rephrase then - please update both sources - I put a lot of effort into migrating this documentation to YAML. If you are only going to document it in one place - I would prefer it to be the YAML because that will be the source of truth at some point (you promised to do the cutover for me like two years ago 😉).

I approved the PR though, feel free to ignore me.

Reply:
Oh ok - I'll do that.

Reply:
The updating both thing, not the ignoring. ;D